### PR TITLE
cart-edit-functionality

### PR DESCRIPTION
**Pull request by SyedAsgar**

### CODE REVIEW

**Review by SyedAsgar — COMMENTED** [review]
image

**Comment by tcs-gohurali** [comment]
image

@SyedAsgar Can you please make these Number datatypes and format your logic respectively? That is how we utilize it in the rest of the project. Once you do that, re-commit and I'll take another look at the conflicts.

File: client/src/app/model.product.ts [hunk 1]
```diff
@@ -7,8 +7,8 @@ export class Product {
 
 export class Data {
     constructor(
-        public id: Number,
-        public name: string,
+        public id: String,
```
Comment (chrisuye) [inline client/src/app/model.product.ts]: make price and quantity numbers

Comment (SyedAsgar) [reply client/src/app/model.product.ts]: Done, Please review. thanks!

**Pull request merged**